src/webviews/apps/plus/graph/GraphWrapper.tsx

```diff
@@ -32,13 +32,43 @@ export interface GraphWrapperProps extends State {
 // Copied from original pushed code of Miggy E.
 // TODO: review that code as I'm not sure if it is the correct way to do that in Gitlens side.
 // I suppose we need to use the GitLens themes here instead.
-export const getCssVariables = (): CssVariables => {
+const getCssVariables = (): CssVariables => {
+    const body = document.body;
+    const computedStyle = window.getComputedStyle(body);
+
+	return {
+        '--app__bg0': computedStyle.getPropertyValue('--color-background'),
+        '--panel__bg0': computedStyle.getPropertyValue('--graph-panel-bg'),
+		'--text-selected': computedStyle.getPropertyValue('--color-foreground'),
+		'--text-normal': computedStyle.getPropertyValue('--color-foreground-85'),
+		'--text-secondary': computedStyle.getPropertyValue('--color-foreground-65'),
+		'--text-disabled': computedStyle.getPropertyValue('--color-foreground-50'),
+		'--text-accent': computedStyle.getPropertyValue('--color-link-foreground'),
+		'--text-inverse': computedStyle.getPropertyValue('--vscode-input-background'),
+		'--text-bright': computedStyle.getPropertyValue('--vscode-input-background'),
+		... getGraphColors(computedStyle),
+    };
+};
+
+const getGraphColors = (computedStyle: CSSStyleDeclaration): CssVariables => {
+	// mixed colors for ref labels (ref label color + graph bg color)
+	// making the ref label color transparent is not an option since ref labels overlap the ref line
+	// and can potentially overlap each other, e.g., when a truncated ref is expanded.
+	const mixedGraphColors: CssVariables = {};
+	for (let i = 0; i < 10; i++) {
+		for (const mixInt of [15,25,45,50]) {
+			mixedGraphColors[`--graph-color-${i}-bg${mixInt}`] = computedStyle.getPropertyValue(`--graph-color-${i}-bg${mixInt}`);
+		}
+	}
+	return mixedGraphColors;
+};
+
+const getStyleProps = ():{cssVariables: CssVariables, themeOpacityFactor: number } => {
 	const body = document.body;
-	const computedStyle = window.getComputedStyle(body);
+    const computedStyle = window.getComputedStyle(body);
 	return {
-		'--app__bg0': computedStyle.getPropertyValue('--color-background'),
-		// note that we should probably do something theme-related here, (dark theme we lighten, light theme we darken)
-		'--panel__bg0': computedStyle.getPropertyValue('--color-background--lighten-05'),
+		cssVariables: getCssVariables(),
+		themeOpacityFactor: computedStyle.getPropertyValue('--graph-theme') === 'dark' ? 1 : 0.5,
```

Comment:
Why not add a new specific variable in `theme.ts` with that logic?

It could be something like in `theme.ts`: 
`bodyStyle.setProperty('--graph-theme-opacity-factor', isLightTheme ? 0.5 : 0);`

And then in `GraphWrapper.jsx`:
`themeOpacityFactor: computedStyle.getPropertyValue('--graph-theme-opacity-factor'),`

Reply:
I thought that it was a little weird if we put something in the CSS that wasn't ever planned to be used by CSS, and thought graph-theme dark/light would be more clean, but can do it this way as well